src/logic/safe/transactions/offchainSigner/index.ts

```diff
@@ -46,10 +46,12 @@ export const tryOffChainSigning = async (
 
       break
     } catch (err) {
-      console.error(err)
       if (err.code === METAMASK_REJECT_CONFIRM_TX_ERROR_CODE) {
         throw err
       }
+      if (isKeystoneError(err)) {
+        throw err
+      }
```

Comment:
This is not doing anything, so I'd suggest to remove it (along with `isKeystoneError`). 
But good to know that we can filter the Keystone errors if we needed to.

Reply:
In the Keystone's interact scenario, user can cancel the QRCode reading or displaying process by clicking the close button and it'll throw an error to the application layer. And (the coding context here)[https://github.com/gnosis/safe-react/blob/dev/src/logic/safe/store/actions/createTransaction.ts#L146] says, if no off chain signing signature was read, it will try to sign again. So in the UI display, it will perform like: 
1. User called Keystone's SDK to sign a transaction. 
2. User canceled signing process. 
3. The Keystone's signing modal appears again.
4. User has to cancel again.

So I throw the error to prevent the "again" process.

Reply:
My bad, I thought there was a default re-throw in that catch. All good.